extension/logstash/lib/logstash/outputs/doris.rb

```diff
@@ -72,6 +73,7 @@ class LogStash::Outputs::Doris < LogStash::Outputs::Base
 
    config :log_progress_interval, :validate => :number, :default => 10
 
+   config :retry_queue_size, :validate => :number, :default => 128
```

Comment:
128 may be too large if the request batch size is large, eg 100MB. So you should limit on queued bytes instead of items.